src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -888,16 +888,16 @@ public function set_status( $status ) {
 		}
 
 
-		$status = wp_update_post( array(
-			'ID' => $this->id,
-			'post_status' => Tribe__Events__Aggregator__Records::$status->{ $status },
-		) );
+		$status = wp_update_post( [
+			'ID'          => $this->id,
+			'post_status' => Tribe__Events__Aggregator__Records::$status->{$status},
+		] );
 
 		if ( ! is_wp_error( $status ) && ! empty( $this->post->post_parent ) ) {
-			$status = wp_update_post( array(
-				'ID' => $this->post->post_parent,
+			$status = wp_update_post( [
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 6c08e2bb43cc99f0794234d74fd90949</sub>